app/javascript/lib/metrics-api.js

```diff
@@ -330,6 +331,22 @@ function getAnalyticsPageName() {
   return getSCPContext().analyticsPageName
 }
 
+/**
+ * gets the tab name for analytics
+ */
+function getTabProperty() {
+  const url = window.location.toString()
+  if (url.includes('step=')) {
+    // if in an upload wizard tab
+    return url.replace(/^(.*?)?step=/, '').trim().replace(/\s/g, '-')
+  } else if (url.includes('#study')) {
+    // if in a study tab
+    return url.replace(/^(.*?)#/, '').trim().replace(/\s/g, '-')
+  } else {
+    return ''
+  }
```

Comment:
Blocker: I believe this logs an empty string as the value for the tab, which seems useless.  The `if (tab)` on line 386 would also be superfluous in this case.

Instead, let's either omit the tab property if it's empty, or remove that `if (tab)` condition.  The former seems preferable.

```suggestion
  } else {
    return null
  }
```

Reply:
An empty string should be false-y same as null ([link](https://frontend.turing.edu/lessons/module-1/js-truthy-falsy-expressions.html?ads_cmpid=6451354298&ads_adid=76255849919&ads_matchtype=b&ads_network=g&ads_creative=517671727591&utm_term=&ads_targetid=dsa-19959388920&utm_campaign=&utm_source=adwords&utm_medium=ppc&ttv=2&gclid=Cj0KCQjw5oiMBhDtARIsAJi0qk15NUZ5cRTIFFBAqeQxnfGzlZ6nxKfsANGro2qJ6XNqVxGs5kubivsaAiNEEALw_wcB)) but I agree that `null` is a much better/clearer return value for this

Reply:
Ah, I didn't know empty string was falsy -- thanks!